config/site_settings.yml

```diff
@@ -885,6 +885,9 @@ rate_limits:
     client: true
     shadowed_by_global: true
     default: 0
+  max_prints_per_hour_per_user:
+    default: 0
```

Comment:
Hmm not sure if this is a good default because it'll raise `Discourse::InvalidAccess`